benchmark.js

```diff
@@ -0,0 +1,27 @@
+const autocannon = require('autocannon')
+const { PassThrough } = require('stream')
+
+function run (url) {
+  const buf = []
+  const outputStream = new PassThrough()
+
+  const inst = autocannon({
+    url,
+    connections: 100,
+    duration: 20
+  })
+
+  autocannon.track(inst, { outputStream })
+
+  outputStream.on('data', data => buf.push(data))
+  inst.on('done', function () {
+    process.stdout.write(Buffer.concat(buf))
+  })
```

Comment:
to avoid garbled output in the console. autocannon seems to write two buffers. one that is a header and one that is the result "body". otherwise it's hard to see which header applies to which body when run in parallel.

Reply:
Perfect, thanks!!